sign/sign_test.go

```diff
@@ -205,38 +205,114 @@ func TestSignImage(t *testing.T) {
 func TestSignFile(t *testing.T) {
 	t.Parallel()
 
+	opts := sign.Default()
+	opts.PrivateKeyPath = "/tmp/private.key"
+	opts.PublicKeyPath = "/tmp/private.pub"
```

Comment:
Let's use `os.MkdirTemp` + `private.{key,pub}` for this and defer it's removal.

Reply:
Sounds good. Will do.

Reply:
Actually we don't need this value to have real data. Its basically a dummy path. In that case do we still need to replace it?

Reply:
Can we ensure that reading/writing to that path in parallel is safe and will not affect the test result? If so, then we can leave it as is but I'd like still have `/tmp` replaced with `os.TempDir` as well as removing the files after the tests. :)

Reply:
In our unit tests we are not interacting with the files. We only check set the value if they are provided but only in using the real implementation we use the files.

Reply:
Got it, thank you for the clarification

Reply:
Changed it to a clear dummy file path for clarity